scss/_root.scss

```diff
@@ -1,4 +1,5 @@
-:root {
```

Comment:
Hi there,

Bringing my stone to this PR. I think that this file should be split this way:

```scss
:root {
  // Keep it as it was before the PR (maybe remove link and code colors)
}

:root, // Because we want it to be the default theme for Bootstrap
[data-bs-theme="light"] {
  // Declare the theme just like the dark one (or using maps like `$theme-colors-text`, ...)
}

[data-bs-theme="dark"] {
  // Keep it as in the actual PR
}
```

I think that this way it would way more understandable for folks. Moreover, it'll decrease the bundle size and avoid redefine all colors everytime we use a `data-bs-theme="light"`. Tbh, I don't really know how a browser process those kind of changes but it might reduce the calculations on user side too.